test/unit/trace-aggregator.test.js

```diff
@@ -333,18 +333,16 @@ describe('TraceAggregator', function () {
       done()
     }
 
-    const testCallback = (err, props) => {
+    // Array iteration is too difficult to slow down, so this steps through recursively
+    txnCreator(0, maxTraces, function testCallback(err, props) {
```

Comment:
so i think these refactors here and across the others could've been solved by reusing one callback function instead of putting it inline. personally this update appeases the sonar rule but makes less readable and still have the same copy pasta cb function

Reply:
this isn't worth the time to refactor the right way